.editorconfig

```diff
@@ -0,0 +1,22 @@
+# http://editorconfig.org
+
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+trim_trailing_whitespace = true
+insert_final_newline = true
+charset = utf-8
+end_of_line = lf
+
+[*.bat]
+indent_style = tab
+end_of_line = crlf
+
+[LICENSE]
+insert_final_newline = false
+
+[Makefile]
+indent_style = tab
+
```

Comment:
What does 'editorconfig' add on top of PEP8?

Reply:
Streamlines how some editors treat spaces, tabs etc. 
Some editors use by default tabs, but spaces are required.

Usually only important in edge cases, but doesn't hurt.

PEP8 is tested via flake8 (see Github workflow)